INTEROP.md

```diff
@@ -20,60 +20,51 @@ imported into an IDE for the purpose of code completion and navigation.
  Interoperability with Swift/Objective-C is provided too and covered in a
 separate document [OBJC_INTEROP.md](OBJC_INTEROP.md).
 
+## Platform libraries ##
+
+ Note that in many cases there's no need to use mechanisms described below, as for APIs available on the
+platform standartized bindings called platform libraries [PLATFORM_LIBS.md](PLATFORM_LIBS.md) could be used.
+For example, POSIX on Linux/macOS platforms, Win32 on Windows platform, or Apple frameworks on macOS/iOS are
+avilable this way.
+
 ## Simple example ##
 
 Build the dependencies and compiler (see `README.md`).
```

Comment:
Do we really need an example which is that hard to play with?

Reply:
Done.